data/src/test/java/com/microsoft/azure/kusto/data/ClientRequestPropertiesTest.java

```diff
@@ -72,21 +109,21 @@ void propertiesToString() throws JsonProcessingException {
     @Test
     @DisplayName("test ClientRequestProperties fromString")
     void stringToProperties() throws JsonProcessingException {
-        String properties = "{\"Options\":{\"servertimeout\":\"01:25:11.111\", \"Content-Encoding\":\"gzip\"},\"Parameters\":{\"birthday\":\"datetime(1970-05-11)\",\"courses\":\"dynamic(['Java', 'C++'])\"}}";
+        String properties = "{\"Options\":{\"servertimeout\":\"00:25:11.111\", \"Content-Encoding\":\"gzip\"},\"Parameters\":{\"birthday\":\"datetime(1970-05-11)\",\"courses\":\"dynamic(['Java', 'C++'])\"}}";
```

Comment:
This is a great example of the previous inconsistencies:
Setting a timeout of >1 day should result in an actual timeout of 1 hour - which it sometimes did, but this is one of the cases when it didn't.